datasets/caner/caner.py

```diff
@@ -0,0 +1,127 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A new corpus of tagged data that can be useful for handling the issues in recognition of Classical Arabic named entities"""
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+import os
+
+import datasets
+
+
+_CITATION = """\
+@article{article,
+author = {Salah, Ramzi and Zakaria, Lailatul},
+year = {2018},
+month = {12},
+pages = {},
+title = {BUILDING THE CLASSICAL ARABIC NAMED ENTITY RECOGNITION CORPUS (CANERCORPUS)},
+volume = {96},
+journal = {Journal of Theoretical and Applied Information Technology}
+}
+"""
+
+_DESCRIPTION = """\
+Classical Arabic Named Entity Recognition corpus as a new corpus of tagged data that can be useful for handling the issues in recognition of Arabic named entities.
+"""
+
+_HOMEPAGE = "https://github.com/RamziSalah/Classical-Arabic-Named-Entity-Recognition-Corpus"
+
+# TODO: Add the licence for the dataset here if you can find it
+_LICENSE = ""
+
+_URL = "https://github.com/RamziSalah/Classical-Arabic-Named-Entity-Recognition-Corpus/archive/master.zip"
+
+
+class Caner(datasets.GeneratorBasedBuilder):
+    """Classical Arabic Named Entity Recognition corpus as a new corpus of tagged data that can be useful for handling the issues in recognition of Arabic named entities"""
+
+    VERSION = datasets.Version("1.1.0")
+
+    def _info(self):
+
+        features = datasets.Features(
+            {
+                "tokens": datasets.Value("string"),
+                "ner_tags": datasets.ClassLabel(
```

Comment:
Since there's only one token per example maybe we should rename the fields `token` and `ner_tag`. What do you think ?

Reply:
yes that makes sense since it's just one token and tag. Will do that now :)